less/index.less

```diff
@@ -4,56 +4,372 @@
+mark,
+audio,
+video {
 	margin: 0;
 	padding: 0;
 	border: 0;
 	font-size: 100%;
 	font: inherit;
 	vertical-align: baseline;
 }
+
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
 	display: block;
 }
+
 body {
 	line-height: 1;
 }
-ol, ul {
+
+ol,
+ul {
 	list-style: none;
 }
-blockquote, q {
+
+blockquote,
+q {
 	quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
+
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
 	content: '';
 	content: none;
 }
+
 table {
 	border-collapse: collapse;
 	border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html,
+body {
+	height: 100%;
+}
+
+//Variables
+@ltgray: lightgray;
+
+//Mixins
+
+.hdr() {
+	font-size: 1.5rem;
+	margin-top: 1.5rem;
+	margin-bottom: 1rem;
+
+}
+
+.flex() {
+	display: flex;
+	flex-direction: row;
+}
+
+.container {
+	width: 880px;
+	margin: 0 auto;
+
+	@media (max-width: 500px) {
+		
+		  width: 95%;
+		  height: 100%;
+		
+	  }
+}
+
+header {
+	text-align: right;
+	margin-top: 1.5rem;
+	margin-bottom: 1.5rem;
+
+	
+	a {
+		color: black;
+		text-decoration: none;
+		font-size: 1.4rem;
+		padding-right: 1.5rem;
+	
+		@media (max-width: 500px) {
+		
+			display: flex;
+			flex-direction: column;
+			align-items: center;
+			border: 1px solid @ltgray;
+			padding: 2rem;
+		}
+
+	}
+}
+
+.main-container {
+	background: @ltgray;
+	padding: 2rem;
+	padding-bottom: 5rem;
+
+	.logo {
+		padding: 2rem;
+	
+	}
+	
+	h1 {
+		font-size: 4rem;
+		margin-bottom: 1rem;
+
+		@media (max-width: 500px) {
+		
+		font-size: 2rem;
+		}
+	
+	}
+
+	h4 {
+		font-size: 1.4rem;
+		border-bottom: 1px solid black;
+		padding-bottom: 1.3rem;
+	}
+
+	p {
+		font-size: 1.5rem;
+		margin-top: 1.2rem;
+		margin-bottom: 1.2rem;
+	}
+
+	.hdr-button {
+		background: darkgray;
+		color: white;
+		font-size: 1rem;
+		padding: .6rem;
+		border-radius: .5rem;
+
+		@media (max-width: 500px) {
+		
+			margin-left: 5rem;
+			font-size: 2rem;
+			}
+		
+	}
 }
 
+.top-content {
+	.flex();
+
+	@media (max-width: 500px) {
+		
+		display: flex;
+		flex-direction: column;
+		}
+
+	.top-text {
+		width: 30rem;
+		padding-right: 2rem;
+
+		@media (max-width: 500px) {
+		
+			width: 29rem;
+			}
+	}
+
+	h2 {
+		.hdr();
+
+		@media (max-width: 500px) {
+		
+			text-align: center;
+			width: 29.4rem;
+			margin-top: 3rem;
+			border-top: 1px solid @ltgray;
+			padding-top: 2rem;
+
+			}
+	}
+
+	p {
+		font-size: 1.2rem;
+
+	
+	}
+
+	h3 {
+		.hdr();
+
+		@media (max-width: 500px) {
+		
+			text-align: center;
+			margin-top: 3rem;
+			border-top: 1px solid @ltgray;
+			padding-top: 2rem;
+			}
+	}
+
+	li {
+		border: 1px solid @ltgray;
+		font-size: 1.3rem;
+		padding: .8rem;
+		width: 25rem;
```

Comment:
Be wary of using rem units for width because it will change depending on the users' set font size.